linter-requirements.txt

```diff
@@ -1,7 +1,6 @@
-black==22.3.0
-flake8==4.0.1
-flake8-import-order==0.18.1
```

Comment:
this was not enabled, plus `isort` is already fixing / enforcing this so I removed it